pkg/agent/openflow/packetin.go

```diff
@@ -52,7 +52,8 @@ const (
 	PacketInReasonNP ofpPacketInReason = 0
 	// PacketInReasonMC shares PacketInReasonNP for IGMP packet_in message. This is because OVS "controller" action
 	// only correctly supports reason 0 or 1. Change to another value after the OVS action is corrected.
-	PacketInReasonMC = PacketInReasonNP
+	PacketInReasonMC        = PacketInReasonNP
+	PacketInReasonSvcReject = PacketInReasonNP
```

Comment:
Not necessary to be addressed in this PR. I'm just wondering if we could change to use a different field than `reason` to distinguish packetIn, so that packetIn won't be picked up by an unrelated handler. Or is it possible for OVS to make `reason` supporting more value? cc @wenyingd